frontend/package.json

```diff
@@ -49,6 +49,7 @@
     "patch-db-client": "file: ../../../patch-db/client",
     "pbkdf2": "^3.1.2",
     "rxjs": "^6.6.7",
+    "src": "^1.1.2",
```

Comment:
Where is this being used?

Reply:
I didn't add anything myself. @dr-bonez could it be something with rebase? Maybe it was there when I started but then got removed but rebase did not sync it? I'll remove it.